### PR TITLE
refactor: use dedicated theme for filament

### PR DESCRIPTION
**Pull request by goga-m**
## Summary
Closes https://app.clickup.com/t/862kc3u3p

It's a potential fix for the issue that is mentioned in the task, an approach that uses a dedicated theme & tailwind config for filament providing the default CSS variables that filament uses, like the `gray` color variants, that are not available in our app's theme. 

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by ItsANameToo** [comment]
@goga-m can you provide details as to why these changes will fix the described issue?

File: public/js/filament/tables/components/table.js
Comment (ItsANameToo) [inline public/js/filament/tables/components/table.js:1]: is this different from `public/js/filament/tables/tables.js`?

**Comment by ItsANameToo** [comment]
I'm still unsure if this will change anything really, as the css error only appears when you switch between branches and the hot reloading somehow doesn't include filament's default theme properly and a subsequent refresh resolves it 🤔

**Comment by crnkovic** [comment]
I kinda agree with @ItsANameToo, plus clearing views seems to fix this issue

**Comment by ItsANameToo** [comment]
closing for now, does not really address the hot reload issue that causes it

**Pull request closed without merging**